### ISSUE TITLE
Exasol Kerberos authentification

### ISSUE THREAD

**Issue by GriGrim** [dbeaver/dbeaver#14737]
#### System information: 
- Operating system (distribution) and version: Windows 10 Pro
- DBeaver version: 7.2.0

#### Connection specification:
- Database name and version: Exasol 6.2.13, Exasol 7.1.2
- Driver name: exajdbc 6.2.9
- kerberoshostname: exasol.example.com
- kerberosrealm: EXAMPLE.COM
- kerberosservicename: exasol
- kerberosusername: danila

#### Describe the problem you're observing:
I cannot connect to Exasol using Kerberos authentication.
I get the following error:
```
Kerberos authentication failed: No valid credentials provided
(Mechanism level: Failed to find any Kerberos tgt)
```

#### Include any warning/errors/backtraces from the logs
```
!ENTRY org.jkiss.dbeaver.model 4 0 2021-11-30 14:44:28.428
!MESSAGE Kerberos authentication failed: No valid credentials provided (Mechanism level: Failed to find any Kerberos tgt)
!SUBENTRY 1 org.jkiss.dbeaver.model 4 0 2021-11-30 14:44:28.428
!MESSAGE Kerberos authentication failed: No valid credentials provided (Mechanism level: Failed to find any Kerberos tgt)
!STACK 0
java.sql.SQLInvalidAuthorizationSpecException: Kerberos authentication failed: No valid credentials provided (Mechanism level: Failed to find any Kerberos tgt)
	at com.exasol.jdbc.AbstractEXAConnection.connectAndLogin(AbstractEXAConnection.java:1991)
	at com.exasol.jdbc.AbstractEXAConnection.setupConnection(AbstractEXAConnection.java:1284)
	at com.exasol.jdbc.AbstractEXAConnection.Connect(AbstractEXAConnection.java:1166)
	at com.exasol.jdbc.AbstractEXAConnection.<init>(AbstractEXAConnection.java:333)
	at com.exasol.jdbc.EXAConnection.<init>(EXAConnection.java:36)
	at com.exasol.jdbc.EXADriver.connect(EXADriver.java:228)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCDataSource.lambda$0(JDBCDataSource.java:178)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCDataSource.openConnection(JDBCDataSource.java:197)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCExecutionContext.connect(JDBCExecutionContext.java:96)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCRemoteInstance.initializeMainContext(JDBCRemoteInstance.java:91)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCRemoteInstance.<init>(JDBCRemoteInstance.java:57)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCDataSource.initializeRemoteInstance(JDBCDataSource.java:111)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCDataSource.<init>(JDBCDataSource.java:106)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCDataSource.<init>(JDBCDataSource.java:95)
	at org.jkiss.dbeaver.ext.exasol.model.ExasolDataSource.<init>(ExasolDataSource.java:103)
	at org.jkiss.dbeaver.ext.exasol.ExasolDataSourceProvider.openDataSource(ExasolDataSourceProvider.java:114)
	at org.jkiss.dbeaver.registry.DataSourceDescriptor.connect(DataSourceDescriptor.java:837)
	at org.jkiss.dbeaver.runtime.jobs.ConnectJob.run(ConnectJob.java:70)
	at org.jkiss.dbeaver.model.runtime.AbstractJob.run(AbstractJob.java:105)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)
```

**Comment by Matvey16** [comment]
@GriGrim Have you tried using a more recent version?

**Comment by GriGrim** [comment]
I've upgraded DBeaver to the version 21.3.1, but the error still remains.

**Comment by Matvey16** [comment]
@GriGrim Can you post a screenshot of driver properties for the connection (with blurred sensitive information ofc)

**Comment by GriGrim** [comment]
![Exasol_connection](https://user-images.githubusercontent.com/2479756/146774639-4b60af24-cb59-4ba3-8e77-e3241c583d0b.png)

**Comment by Matvey16** [comment]
Based on the error message it seems like there is a problem with krb5.conf file. Can you check that everything is correct there?

**Comment by GriGrim** [comment]
I didn't create neither krb5.conf, nor krb5.ini file.
Before I upgraded Exasol JDBC driver to version 7.1 kerberos connection was working fine. Now I rolled back Exasol JDBC driver to 6.2, but kerberos connection is still failing. My colleagues who didn't upgrade Exasol JDBC driver still can connect without any problems.
Interesting is that ExaPlus client, which is using the same JDBC driver, is performing kerberos connection successfully.
